README.md

```
@@ -12,19 +12,41 @@
 </div>
 
-### Tools
-1. [Memotron](https://memotron.app) - Your memory atlas
-2. [Pointron](https://pointron.app) - Your focus haven
-3. [Nuclues](https://nucleus.to) - Your digital harmony
+# Tools
+The below is the list of tools built using Tidigit library.
 
-More coming soon...
+| Name | Website | Description | Topics
+|------|---------|-------------|-------|
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_only; Actual: leading_and_trailing; Unexpected trailing pipe

(MD055, table-pipe-style)